shared/src/iosMain/kotlin/co/touchlab/kampstarter/NativeCoroutineAdapter.kt

```diff
@@ -0,0 +1,53 @@
+package co.touchlab.kampstarter
+
+import co.touchlab.kampstarter.db.Breed
+import co.touchlab.kampstarter.models.BreedModel
+import co.touchlab.kampstarter.models.ItemDataSummary
+import co.touchlab.kermit.*
+import kotlinx.coroutines.*
+import kotlinx.coroutines.flow.*
+import co.touchlab.stately.ensureNeverFrozen
+
+
+class NativeCoroutineAdapter(
```

Comment:
I know we used it previously but this name feels a bit out-of-place now. Maybe `ViewModel` or `NativeViewModel` since it pretty well parallels the android viewmodel's role.

Reply:
Makes sense. Updated